data/db-schema/src/main/resources/net/corda/db/schema/vnode-vault/migration/vnode-vault-creation-v5.1.xml

```diff
@@ -12,5 +12,10 @@
                 <constraints nullable="false" primaryKey="true"/>
             </column>
         </createTable>
+
+        <modifyDataType
+                columnName="reason"
+                newDataType="varchar(1000)"
```

Comment:
Maybe we should change it to` TEXT`?

(If we do keep it as varchar, we need to add a code to trim the reason in the runtime code)

Reply:
I think you are right. Text is better suited for long messages and we possibly have lengthy reasons at times.